plyer/tests/test_battery.py

```diff
@@ -196,6 +197,100 @@ def percentage():
         return float(percentage.replace(',', '.'))
 
 
+class MockedIOReg(object):
+    '''
+    Mocked object used instead of Apple's ioreg.
+    '''
+    values = {
+        "MaxCapacity": "5023",
+        "CurrentCapacity": "4222",
+        "IsCharging": "No"
+    }
+
+    output = """+-o AppleSmartBattery  <class AppleSmartBattery,\
```

Comment:
Use bytes for `output` so that the `decode('utf-8')` in the platform implementation doesn't fail.

https://docs.python.org/2/library/subprocess.html#subprocess.Popen.communicate
https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate